### PR TITLE
[Release] Bump to 0.2.0

### PR DESCRIPTION
**Pull request by odesenfans**
Updated the README to detail the support for both daemons.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,9 +4,17 @@
 
 > The [libp2p daemon](https://github.com/libp2p/go-libp2p-daemon) bindings for Python
 
-Have aligned with the go-client now!
+Provides a client library to interact with the official libp2p daemons.
+Supports the Go and JavaScript daemons.
```
Comment (mhchia) [inline README.md:8]: Nit: I'd suggest we add links to the repos.
```suggestion
Supports the [Go](https://github.com/libp2p/go-libp2p-daemon) and [JavaScript](https://github.com/libp2p/js-libp2p-daemon) daemons.
```

**Review by mhchia — APPROVED** [review]
LGTM! Exciting to see CD happening later:)

**Comment by odesenfans** [comment]
Worked like a charm! https://pypi.org/project/p2pclient/

**Comment by mhchia** [comment]
Great work. Thanks for making the project ready.

**Comment by odesenfans** [comment]
No worries, thanks for the good work on this library it's been a real time saver so far!

**Pull request merged**